modules/ticket_selector/DisplayTicketSelector.php

```diff
@@ -88,13 +108,19 @@ public function __construct($iframe = false)
             'FHEE__EED_Ticket_Selector__display_ticket_selector__time_format',
             get_option('time_format')
         );
+        $this->admin_roles = apply_filters(
+            'FHEE__EED_Ticket_Selector__display_ticket_selector__admin_roles',
+            ['administrator']
+        );
+        $this->current_user = wp_get_current_user();
+        $this->current_user_is_admin = ! empty(array_intersect($this->admin_roles, $this->current_user->roles));
```

Comment:
you are correct...
I debated about doing something similar to what's in [\core\domain\services\admin\events\editor\EventManagers](https://github.com/eventespresso/event-espresso-core/blob/dev/core/domain/services/admin/events/editor/EventManagers.php) but tried to cheat get away with just checking the role.

so.. um... ya... you caught me 😕 


I think I'll have to extract some of the logic in that class I linked to so that it's not getting duplicated